### ISSUE TITLE
CORS policy is not respected

### ISSUE THREAD

**Issue by djuarezg** [sourcegraph/sourcegraph#969]
- **Sourcegraph version:** 2.13.2
- **Platform information:** Running on CentOS7, using Chrome extension, on Docker 1.13.1

#### Steps to reproduce:

1. Run your own Sourcegraph local instance
2. Configure the extension to connect to this instance
3. Configure Sourcegraph as in https://docs.sourcegraph.com/integration/browser_extension#configuring-the-sourcegraph-instance-to-use to set up CORS policy origins pointing to our private Gitlab instance as here:

```
{
  "auth.providers": [
    {
      "type": "builtin"
    }
  ],
  "disablePublicRepoRedirects": true,
  "maxReposToSearch": 50,
  "corsOrigin": "https://<our_private+instance>",
  ...
```

#### Expected behavior:
As in the past, be able to get the extension to work while opening files that belong to enabled and cloned repositories.
#### Actual behavior:
Even if the project is enabled, cloned and the Gitlab token has access to the project:

Chrome console output:
```
Access to XMLHttpRequest at 'https://<my_sourcegraph_local_instance>/.api/graphql?Configuration' from origin 'chrome-extension://dgjhfomjieaadpoljlnidmbgkdffpack' has been blocked by CORS policy: Request header field authorization is not allowed by Access-Control-Allow-Headers in preflight response.
```

**Comment by sqs** [comment]
Reopening for @nicksnyder to ship a new patch release of 2.13 that incorporates this fix.

**Comment by nicksnyder** [comment]
This is fixed in 2.13.3
